make_bfb.sh

```
@@ -54,35 +55,98 @@ buildbfb() {
+    # If the default bfb, capsule and infojson weren't provided on
+    # the command line then we'll pull them from the container
+    if [ -z "${default_bfb}${capsule}${infojson}" ]; then
+        CID=$(podman run -d "${coreos_bfb_container}" sleep infinity)
+        podman cp $CID:/lib/firmware/mellanox/boot/default.bfb $WDIR/default.bfb
+        podman cp $CID:/lib/firmware/mellanox/boot/capsule/boot_update2.cap $WDIR/boot_update2.cap
+        podman cp $CID:/usr/opt/mellanox/bfb/info.json $WDIR/info.json
+        podman stop $CID
+        podman rm $CID
+    else
+        # Otherwise we'll use the ones provided by the user.
+        if [ ! -f "${default_bfb}" -o ! -f "${capsule}" -o ! -f "${infojson}" ]; then
```

Shellcheck (0.10.0)

[warning] Prefer [ p ] || [ q ] as [ p -o q ] is not well defined.

(SC2166)